oauth2-authorization-server/src/main/resources/org/springframework/security/oauth2/server/authorization/oauth2-authorization-schema-postgres.sql

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CREATE TABLE oauth2_authorization (
```

Comment:
Regarding your comment:

> Do you think it's still necessary to have a different file where the `token` columns are defined as `text`?

On second thought, I think it's redundant so let's remove. Can you put a comment in `oauth2-authorization-schema.sql` along the lines...."NOTE: For MySQL and PostgreSQL, change the blob columns types to text"